components/camera/client.go

```diff
@@ -78,7 +78,7 @@ func (c *client) Stream(
 	ctx context.Context,
 	errHandlers ...gostream.ErrorHandler,
 ) (gostream.VideoStream, error) {
-	cancelCtxWithMIME := gostream.WithMIMETypeHint(c.cancelCtx, gostream.MIMETypeHint(ctx, utils.MimeTypeRawRGBALazy))
+	cancelCtxWithMIME := gostream.WithMIMETypeHint(c.cancelCtx, gostream.MIMETypeHint(ctx, ""))
 	streamCtx, stream, frameCh := gostream.NewMediaStreamForChannel[image.Image](cancelCtxWithMIME)
```

Comment:
I would say just remove the entire line, and just do 
`streamCtx, stream, frameCh := gostream.NewMediaStreamForChannel[image.Image](c.cancelCtx)`

Reply:
Oh actually never mind, you're right to do what you're doing. Because the context in Stream might have a MIMETypeHint itself.